yarn-project/acir-simulator/src/avm/opcodes/bitwise.ts

```diff
@@ -9,16 +8,18 @@ export class And extends Instruction {
   static type: string = 'AND';
   static numberOfOperands = 3;
```

Comment:
Lets add a static hasTag to this and other instructions, I am ok with that being ontop of a stack, or creating an issue for it to be later in the day.

Reply:
Created issue [4218](https://github.com/AztecProtocol/aztec-packages/issues/4218). I explain some concerns there.